code/game/objects/items/weapons/grenades/flashbang.dm

```diff
@@ -29,23 +29,21 @@
 	if(M.weakeyes)
 		M.visible_message("<span class='disarm'><b>[M]</b> screams and collapses!</span>")
 		M << "<span class='userdanger'><font size=3>AAAAGH!</font></span>"
-		M.Weaken(15) //hella stunned
-		M.Stun(15)
+		M.Weaken(5) //hella stunned
+		M.Stun(5)
 		M.adjust_eye_damage(8)
 
 	if(M.flash_eyes(affect_silicon = 1))
-		M.Stun(max(10/distance, 3))
-		M.Weaken(max(10/distance, 3))
+		M.Stun(1)
+		M.confused += (max(10/distance, 3))

```

Comment:
And maybe make the confusion from the deafness part? Though everyone seems mad enough that this probably isnt worth the trouble